packages/backend/src/open_payments/receiver/service.test.ts

```diff
@@ -248,14 +268,12 @@ describe('Receiver Service', (): void => {
         }
 
         beforeEach(async (): Promise<void> => {
-          paymentPointer = await createPaymentPointer(deps)
-          incomingPayment = await createIncomingPayment(deps, {
-            paymentPointerId: paymentPointer.id,
-            incomingAmount: {
-              value: BigInt(5),
-              assetCode: paymentPointer.asset.code,
-              assetScale: paymentPointer.asset.scale
-            }
+          paymentPointer = mockPaymentPointer({
+            authServer
+          })
+          incomingPayment = mockIncomingPayment({
+            id: `${paymentPointer.id}/incoming-payments/${uuid()}`,
+            paymentPointer: paymentPointer.id
```

Comment:
I thought it made more sense to mock the payment pointer and incoming payment here, since we are fetching remote incoming payments, and do not need create any locally to test this